src/bun.js/modules/NodeBufferModule.h

```diff
@@ -1,3 +1,7 @@
+#pragma once
```

Comment:
i think these files were used in a weird way where this root.h isnt needed but i think this is reasonable to have nonetheless since it compiles